drivers/usb/udc/udc_stm32.c

```diff
@@ -912,6 +913,7 @@ static const struct udc_stm32_config udc0_cfg = {
 	.pma_offset = USB_BTABLE_SIZE,
 	.ep0_mps = EP0_MPS,
 	.ep_mps = EP_MPS,
+	.speed_idx = DT_ENUM_IDX(DT_DRV_INST(0), maximum_speed),
```

Comment:
maximum_speed property is not available from all DT.
In old driver, property is check before access it:
https://github.com/zephyrproject-rtos/zephyr/blob/c0be51df2fb83f2e772d74b73e10350662af12b8/drivers/usb/device/usb_dc_stm32.c#L70-L72

Reply:
```suggestion
	.speed_idx = DT_ENUM_IDX_OR(DT_DRV_INST(0), maximum_speed, low-speed),
```